examples/helloworld/hello_world.cc

```diff
@@ -0,0 +1,108 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <time.h>
+#include <cstdlib>
+#include <iostream>
+#include <memory>
+#include <string>
+
+#include "absl/strings/string_view.h"
+#include "absl/time/clock.h"
+#include "absl/time/time.h"
+#include "opencensus/exporters/stats/stdout/stdout_exporter.h"
+#include "opencensus/exporters/trace/stdout/stdout_exporter.h"
+#include "opencensus/stats/stats.h"
+#include "opencensus/trace/sampler.h"
+#include "opencensus/trace/span.h"
+
+ABSL_CONST_INIT const absl::string_view kFrontendKey = "my.org/keys/frontend";
```

Comment:
Please wrap everything besides main() in an anonymous namespace.

Reply:
Done.